src/EPiServer.SocialAlloy.ExtensionData/Membership/MembershipModeration.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using EPiServer.Social.Groups.Core;
 
-namespace EPiServer.SocialAlloy.Web.Social.Models.Moderation
+namespace EPiServer.SocialAlloy.ExtensionData.Membership
```

Comment:
This is a separate projected just for social daily extensiondata. All other extensiondata has been moved to this project.